app/[locale]/profile/layout.tsx

```diff
@@ -14,9 +14,9 @@ const ProfileLayout = async ({
   const session = await getServerAuthSession();
 
   return session ? (
-    <section>
+    <>
       <div className="bg-gray-100 mt-11 flex h-full gap-2 p-16">{children}</div>
-    </section>
+    </>
```

Comment:
Just remove the tags entirely, they serve no purpose. You only need them when wrapping multiple elements but don't want to add a wrapper tag.

Reply:
alright, shifting wrapper component to page.tsx

Reply:
pushed